src/lcd.c

```diff
@@ -327,12 +312,18 @@ static inline void LCD_WriteData(uint32_t databyte, uint8_t chipnum) {
 	FIO1PIN;*/
 }
 
+#define LCD_ON (0x3f)
+#define LCD_RESET_X (0xb8)
+#define LCD_RESET_Y (0x40)
+#define LCD_RESET_STARTLINE (0xc0)
+
+
 void LCD_Init(void) {
 	FIO1DIR = 0xff0000; // Data pins output
-	LCD_WriteCmd(0x3f); // Display on
-	LCD_WriteCmd(0x40); // Set Y address to 0
-	LCD_WriteCmd(0xb8); // Set X page 0
-	LCD_WriteCmd(0xc0); // Set display start line to 0
+	LCD_WriteCmd(LCD_ON);
+	LCD_WriteCmd(LCD_RESET_X);
+	LCD_WriteCmd(LCD_RESET_Y);
+	LCD_WriteCmd(LCD_RESET_STARTLINE);
```

Comment:
I think this coding style helps against out of sync comments. These might have to end up in the .h file?